src/mol-model/structure/structure/structure.ts

```diff
@@ -232,7 +232,12 @@ class Structure {
 
     get interUnitBonds() {
         if (this.state.interUnitBonds) return this.state.interUnitBonds;
-        this.state.interUnitBonds = computeInterUnitBonds(this, { ignoreWater: !this.dynamicBonds });
+        if (this.parent && this.state.dynamicBonds === this.parent.state.dynamicBonds && this.parent.state.interUnitBonds?.edgeCount === 0) {
+            // no need to compute InterUnitBonds if parent's ones are empty
+            this.state.interUnitBonds = new InterUnitBonds(new Map());
+        } else {
```

Comment:
This seems to be missing the condition ``this.parent.state.interUnitBonds !== void 0``, which is not the same as ``this.parent.state.interUnitBonds?.edgeCount === 0`` (e.g. it is possible to trigger the computation on the child structure without doing the same for the parent.

Reply:
indeed, thanks